src/admin/redux/modules/email/reducer.js

```diff
@@ -97,7 +97,7 @@ const reducer : Reducer<State, Action>
 
 export default reducer
 
-const sort_by_date = r_compose(reverse, sortBy(prop('created_at')), map(over(lensProp('created_at'), Date.parse)))
+const sort_by_date = pipe([map(over(lensProp('created_at'), Date.parse)), sortBy(prop('created_at')), reverse])
```

Comment:
Agreed. It's not worth the overhead of all the noise for a linter, but I'd be OK with eslint configured to only check for unused variables (or an equivalent if we can do it without `eslint`)